META.json

```diff
@@ -49,7 +49,8 @@
             "Test::Output" : "1.03",
             "Test::Simple" : "1.001002",
             "Test::Spec" : "0.49",
-            "Test::TempDir::Tiny" : "0.016"
+            "Test::TempDir::Tiny" : "0.016",
+            "Test::Which" : "1.27"
```

Comment:
It does not look like `Test::Which`  is directly used in  the new test, we can probably remove this change. 

Actually I pretty much just depend on `mbtiny` to generate META.json / META.yml for me these days...

Reply:
Interesting... CI failed at  missing Test::Switch. I guess I'll need to tweak how I use mbtiny a little bit to accommodate this addition.